.eslintrc.cjs

```diff
@@ -127,6 +128,12 @@ module.exports = {
 		"@typescript-eslint/no-unused-vars": ["error", { caughtErrors: "all" }],
 		"import/extensions": ["error", "ignorePackages"],
 		"no-only-tests/no-only-tests": "error",
+		"n/no-missing-import": [
+			"error",
+			{
+				allowModules: ["template-typescript-node-package"],
+			},
+		],
```

Comment:
OH I almost forgot, these changes need to be applied to https://github.com/JoshuaKGoldberg/template-typescript-node-package/blob/0ac81b32735afcebb9d3489f860427a2c9bfd5d3/src/hydrate/steps/writing/creation/rootFiles.ts! Filed #546.

I'll go ahead and make this change because I don't want to keep rudely blocking you 😄

Reply:
Oh, I am not aware of that.
Haven't touched the hydrate section yet.
Thanks for doing that 💪